hooks/commit-msg.d/check-if-message-accords-with-style-guide.sh

```diff
@@ -6,7 +6,7 @@
 # characters length. The body is considered everything until the end of the message.
 pattern="^(feat|fix|docs|style|refactor|test|chore):[ ](.{1,50})("$'\n'"("$'\n'"[^"$'\n'"]{0,72})+)?$"
 
-message=$(cat "${1}")
+message=$(grep --invert-match "^#" < "${1}")
```

Comment:
Why not just use `-v`? I think this is too verbose.

Reply:
@likeanowl for documentation purposes it is better to use full syntax in scripts rather than short one.

Reply:
I can agree with this, but first link when googling "grep invert match" will suggest a short argument$ do we really want to be so verbose?

Reply:
@likeanowl, you obviously will use a short syntax when you want to run a single command. But in scripts, it is better to be as verbose as you can, for the same reason we do not use single letter variables when writing in other languages - readability.

Reply:
@DenisVerkhoturov OK.

Reply:
![photo_2019-10-17_20-33-34](https://user-images.githubusercontent.com/37806636/67033172-c8663d80-f11d-11e9-8fce-0ef248494641.jpg)

Reply:
@andreibelov I still think that `--invert-match` is absolutely OK here, since this script is quite simple and this argument could be googled very easy.